Editor/UI/Drawers/BehaviorInstantiatiorDrawer.cs

```diff
@@ -24,6 +24,11 @@ public override Rect Draw(Rect rect, object currentValue, Action<object> changeV
                 TestableEditorElements.DisplayContextMenu(options);
             }
             EditorGUI.EndDisabledGroup();
+            
+            if (EditorDrawingHelper.DrawHelpButton(ref rect))
```

Comment:
This button in my opinion seems like over communication and more disturbing than helpful. What is confusing about "Add Behavior"?